tls/s2n_x509_validator.h

```diff
@@ -60,6 +61,8 @@ struct s2n_x509_validator {
     uint16_t max_chain_depth;
     STACK_OF(X509) *cert_chain_from_wire;
     int state;
+    STACK_OF(X509_CRL) *crl_stack;
```

Comment:
It looks like the crl_stack is only used after the crl callback completes. Why do we need to store it on the validator? Couldn't we just build it at the point where we need it?

Reply:
Yes, true. I removed this from the validator and create it in `s2n_x509_validator_verify_cert_chain` now instead. 91bfce152eb1068dda6897f39daf4932fcf21ea1